hw_5/circle.py

```diff
@@ -0,0 +1,22 @@
+class Circle:
+    def __init__(self, x, y, radius):
+        self.x = x
+        self.y = y
+        self.radius = radius
+
+    def contains(self, instance):
+        if instance.x ** 2 + instance.y ** 2 <= self.radius ** 2:
+            return True
+        else:
+            return False
```

Comment:
Це можливо спростити 
```suggestion
 return instance.x ** 2 + instance.y ** 2 <= self.radius ** 2
```
Але як метод враховує кординати самого кола?